jumanji/env.py

```diff
@@ -15,14 +15,22 @@
 """Abstract environment class"""
 
 import abc
-from typing import Any, Generic, Tuple, TypeVar
+from typing import Any, Generic, Protocol, Tuple, TypeVar
 
+import chex
 from chex import PRNGKey
```

Comment:
NIT: In this case, can we remove this line and use `chex.PRNGKey` in this module? It's more explicit.

Reply:
Done.